.npmignore

```diff
@@ -0,0 +1,9 @@
+.tmp/
+spec/
+src/
+Gruntfile.coffee
+coffeelint.json
+.travis.yml
+
+.idea/
+cukefarm.iml
```

Comment:
See [here](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package) for documentation on what npm ignores by default.